mentat/conversation.py

```diff
@@ -92,18 +92,11 @@ async def display_token_count(self):
             )
         else:
             self.max_tokens = context_size
-        if context_size and tokens > context_size:
```

Comment:
On line 80 (can't comment inline there) `max_tokens` is set to 0. Is there a reason it isn't set to `context_size`? I think it might not have been updated when we changed the meaning of the field from allowed auto_tokens.